Assets/UI/Popups/policyreminderpopup.xml

```diff
@@ -28,7 +28,7 @@
         </Stack>
 
        <!-- Confirm Button -->
-        <GridButton ID="ChangeGovernmentButton" Size="180,35" Anchor="C,B" Offset="0,25" String="LOC_RESEARCH_COMPLETE_CIVIC_BUTTON" Style="MainButton"/>
+        <GridButton ID="ChangeGovernmentButton" Size="190,38" Anchor="C,B" Offset="0,25" String="LOC_RESEARCH_COMPLETE_CIVIC_BUTTON" Style="MainButton"/>
```

Comment:
I'm not sure, the graphic of the button is made for an height of `40-41`, otherwise it as some artifacts. As for the width, I'll need to test in other languages if it fit.